interfaces/builtin/audio_playback.go

```diff
@@ -138,6 +156,7 @@ func (iface *audioPlaybackInterface) StaticInfo() interfaces.StaticInfo {
 	return interfaces.StaticInfo{
 		Summary:              audioPlaybackSummary,
 		ImplicitOnClassic:    true,
+		ImplicitOnCore:       false,
```

Comment:
We shouldn't need this, since the field is implicitly zero initialised. We don't explicitly initialise these fields to false in any of the other interfaces, so shouldn't do it here.